docs/docs/getting-started/index.html

```diff
@@ -17,7 +17,7 @@
 </head>
```

Comment:
I think this is a generated file.

Reply:
Thanx.
I closed this PR and I'll request new PR again.